index.html

```diff
@@ -0,0 +1,174 @@
+                <a class="portfolio-link-a-nav" href="#about">Join</a>
+            </li>
+            <li class="portfolio-link">
+                <a class="portfolio-link-a-nav" href="#">Sponsor</a>
+            </li>
+            <li class="portfolio-link">
+                <a class="portfolio-link-a-nav" href="#">News</a>
+            </li>
+            <li><a class="summit" href="#">The summit</a></li>
+        </ul>
+        <div class="menu-container">
+            <span class="material-symbols-outlined">
+                menu
+            </span>
+        </div>
+    </header>
+    <section id="home" class="main-section">
+        <header class="nav">
+            <div class="menu-container">
+                <span id="hamburger" class="material-symbols-outlined">
+                    menu
+                </span>
+                <img src="img/logo.png" alt="" srcset="">
+            </div>
+        </header>
+        <!-- Mobile Menu -->
+        <div class="mob-menu">
+            <div>
+                <button id="close">&#9747;</button>
+            </div>
+            <ul class="mob-list">
+                <li class="portfolio-link">
+                    <a class="portfolio-link-a new-link" href="index.html">Home</a>
+                </li>
+                <li class="portfolio-link">
+                    <a class="portfolio-link-a new-link" href="about.html">About</a>
+                </li>
+                <li class="portfolio-link">
+                    <a class="portfolio-link-a new-link" href="#contact-section">Program</a>
+                </li>
+                <li class="portfolio-link">
+                    <a class="portfolio-link-a new-link" href="#contact-section">Join</a>
+                </li>
+                <li class="portfolio-link ">
+                    <a class="portfolio-link-a new-link" href="#contact-section">News</a>
+                </li>
+            </ul>
+        </div>
+        <!-- ================First Section Home page =========================-->
+        <article class="main-section-article">
+            <h1>Sharm el-Sheikh Climate Change Conference</h1>
+            <h2>A defining moment in the fight against climate change 2022</h2>
+            <p class="article-1">Science has established beyond doubt that the window for action is closing rapidly. In
+                November 2022,
+                Egypt will host the 27th Conference of the Parties of the UNFCCC (COP27) in Sharm El-Sheikh, with a view
+                to building on previous successes and paving the way for future ambition.
+                A golden opportunity for all stakeholders to rise to the occasion and tackle effectively the global
+                challenge of climate change facilitated by Egypt on the African continent.</p>
+            <p class="date">6 November to 18 November 2022</p>
+            <p class="place">@ Sharm El Sheikh International Convention Centre</p>
+        </article>
```

Comment:
- [ ] Kindly fix the overlapping text for the mobile version seen in the following image.
![image](https://user-images.githubusercontent.com/92215878/192031981-8a993a41-23a6-4d36-88f6-1f4dd4a4781c.png)

Reply:
![fix-overlapping](https://user-images.githubusercontent.com/79546797/192043542-646e64c1-fbbf-445e-971f-6dda23c0b0b4.png)
@jasonfelice